src/App.tsx

```diff
@@ -1,61 +1,112 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function findUserById(id: number) {
```

Comment:
Consider using `Array.prototype.find()` instead of `Array.prototype.filter()` to find a user by id. It will make the code more efficient. 🚀